### ISSUE TITLE
Add identifiers to entity records (from 1822)

### ISSUE THREAD

**Issue by gigamorph** [project-lux/lux-frontend#44]
Problem description:  We are adding more identifiers to Works records on #944 .  There are also identifiers that need to be identified and shown on People, Place, Concept and Events pages.

Expected Behavior: Ensure all identifiers that should be shown to users are displayed. 

Needed 

- [ ] Confirmation of missing identifiers for People - Kelly
 - ORCID
- [ ] Confirmation of missing identifiers for Places - Kelly
- [ ] Confirmation of missing identifiers for Concepts - Kelly
- [ ] Confirmation of missing identifiers for Events - Kelly
- [ ] Confirm UAT links for above - Heather
- [ ] Mockups to confirm where these are shown - is ordering, as provided in data? 
- People
https://lux.collections.yale.edu/view/person/783e7e6f-6863-4978-8aa3-9e6cd8cd8e83
- Places
https://lux.collections.yale.edu/view/place/0f7662ab-35d1-4b90-ba96-66b3663cc512
- Concepts
https://lux.collections.yale.edu/view/concept/f4dc7026-690c-452d-b976-9fc939591b6a
- Events
https://lux.collections.yale.edu/view/activity/4d1f9ff3-a42f-46f4-9da1-4295e169d130

**Comment by roamye** [comment]
Comments from old tix: 
kd736  -
"@prowns this can't be worked on yet. the inspiring ticket was for Works missing IDs, which exist in the data and aren't being shown, but none of these entities have identifier data not being shown. We intend to add ORCIDs (and perhaps other identifiers) for some people but they're not there yet."

**Comment by roamye** [comment]
@kkdavis14 were the ORCID's (and other identifiers) added? or are we still waiting on this?

**Comment by kkdavis14** [comment]
@roamye there aren't any examples yet. We may be better served by closing this ticket and opening one when we have an example.

**Comment by roamye** [comment]
Per UAT leaving this opened @kkdavis14.

**Comment by roamye** [comment]
per tick forming this is not blocked and will be worked on in the future.

**Comment by zachjschwartz** [comment]
TF 5/5: fine to just leave in future work for the moment.
